tensorflow/python/ops/nn_impl.py

```diff
@@ -333,8 +335,12 @@ def l2_normalize(x, dim, epsilon=1e-12, name=None):
     A `Tensor` with the same shape as `x`.
   """
   with ops.name_scope(name, "l2_normalize", [x]) as name:
+    if dim is not None:
```

Comment:
could you use deprecation.deprecated_argument_lookup() (it provides this functionality)

Reply:
Thanks @aselle. Updated.